heuristics.py

```diff
@@ -0,0 +1,117 @@
+import random
+
+from kubernetes import client, config, watch
+
+from aws_metrics import get_volatility_metrics
+
+SPOT_LABEL = 'spot-instance'
+VOLATILITY_SPAN = 168
+
+
+config.load_kube_config('./config-spark-on-cat')
+v1 = client.CoreV1Api()
+
+def nodes_available():
+    ready_nodes = []
+    for node in v1.list_node().items:
+        for status in node.status.conditions:
+            if status.status == "True" and status.type == "Ready":
```

Comment:
how do you know if a node has capacity for your pod?

Reply:
Did not add this check since it would be duplicating behavior implemented in the default scheduler, which will be leveraged when implemented as a scheduler extension.